pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -31,17 +31,92 @@ local zotero = require('zotero')
 -- -- global state -- --
 local config = {
   client = 'zotero',
-  scannable_cite = false
+  scannable_cite = false,
+  csl_style = '',
+  transferable = false
 }
 
+-- -- -- bibliography marker generator -- -- --
+function zotero_docpreferences_odt(csl_style)
+  return string.format(
+    '<data data-version="3" zotero-version="5.0.89">'
+      .. '   <session id="OGe1IYVe"/>'
+      .. '   <style id="http://www.zotero.org/styles/%s" locale="en-US" hasBibliography="1" bibliographyStyleHasBeenSet="0"/>'
```

Comment:
> I don't know anything about this header TBH, but shouldn't `bibliographyStyleHasBeenSet` be "1" here since we're supplying the style?

I don't have an answer to this.  When I unit-tested, I wasn't running in to any problems.  (You can look at the videos I shared with you earlier)